modules/installation-and-upgrade/pages/container-deployment/suma/server-migration-suma.adoc

```diff
@@ -60,6 +59,11 @@ After the migration, it will be necessary to manually update the DHCP and DNS re
 
 === Initial Preparation on the Old 4.3 Server
 
+[IMPORTANT]
+====
+The migration can take a very long time depending on the amount of data that needs to be replicated. To reduce downtime it is possible to run the migration multiple times in a process of "initial replication", "re-replication", "final replication and switch over" while all the services on the source can stay up and running and only during the final migration the processes on the source need to be stopped. For all "non-final" replications add the parameter [command]``--prepare`` needs to be added to to prevent the automatic stopping the services on the source server. I.e. use [command]``mgradm migrate podman <oldserver.fqdn> --prepare`` to prevent the automatic stopping the services on the source.
```

Comment:
```suggestion
The migration can take a very long time depending on the amount of data that needs to be replicated. To reduce downtime it is possible to run the migration multiple times in a process of "initial replication", "re-replication", "final replication and switch over" while all the services on the source can stay up and running and only during the final migration the processes on the source need to be stopped. For all "non-final" replications the parameter [command]``--prepare`` needs to be added to to prevent the automatic stopping the services on the source server. I.e. use [command]``mgradm migrate podman <oldserver.fqdn> --prepare`` to prevent the automatic stopping the services on the source.
```